static/js/mass-update.js

```diff
@@ -95,19 +102,18 @@ $(document).ready(function() {
                 }
             });
         }
+
         if (totalCount === 0) {
             return false;
         }
-        var params = $.param({
-            toUpdate: updateArr.join('|'),
-            toRefresh: refreshArr.join('|'),
-            toRename: renameArr.join('|'),
-            toSubtitle: subtitleArr.join('|'),
-            toDelete: deleteArr.join('|'),
-            toRemove: removeArr.join('|'),
-            toMetadata: metadataArr.join('|')
+        if (updateArr.length + refreshArr.length + renameArr.length + subtitleArr.length + deleteArr.length + removeArr.length + metadataArr.length === 0) {
+            return false;
+        }
+        const url = $('base').attr('href') + 'manage/massUpdate';
+        const params = 'toUpdate=' + updateArr.join('|') + '&toRefresh=' + refreshArr.join('|') + '&toRename=' + renameArr.join('|') + '&toSubtitle=' + subtitleArr.join('|') + '&toDelete=' + deleteArr.join('|') + '&toRemove=' + removeArr.join('|') + '&toMetadata=' + metadataArr.join('|');
+        $.post(url, params, function() {
+            location.reload(true);
```

Comment:
Is it the best idea to refresh since we may end up sending the request twice because of the form?

Reply:
Let me stop you guys right there. I'm not interested in creating all kind of fancy features in our "old" frontend (like utilizing the websockets for this). I don't want to spend my time on that. If you think that the current PR doesn't hold enough value as is, then i'd say remove it. As I got lot's of things to work on (like bootstrapping the config pages or finishing super's binsearch provider).

Keep in mind, the massUpdate/Edit page in practice is used once by user, and then the user moves on in his/her use flow. Also I still don't see how we can send the request twice **by accident**.